classes/page_group.php

```diff
@@ -54,6 +83,27 @@ protected function set_fuzzydurationcounts(array $data) {
         $this->raw_set('fuzzydurationcounts', json_encode($data, JSON_FORCE_OBJECT));
     }
 
+    /**
+     * Called after the page group is created in the DB.
+     */
+    public function after_create() {
+        $this->after_update(true);
+    }
+
+    /**
+     * Called after the DB is updated.
+     *
+     * @param bool $result
```

Comment:
Can we use the parent's phpdoc for the param here, as it is a bit more descriptive.
```suggestion
     * @param bool $result Whether or not the update was successful.
```